dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderFactoryImpl.java

```diff
@@ -107,88 +101,89 @@ protected Folder find(String folderInode) throws DotDataException {
 		return folder;
 	}
 
-
+	/**
+	 *
+	 * @param folder
+	 * @return
+	 * @throws DotStateException
+	 * @throws DotDataException
+	 * @deprecated use {@link #getSubFoldersTitleSort(Folder)}
+	 */
 	@SuppressWarnings("unchecked")
 	@Override
+	@Deprecated
 	protected java.util.List<Folder> getSubFolders(Folder folder) throws DotStateException, DotDataException {

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)